src/components/Header/index.tsx

```diff
@@ -0,0 +1,183 @@
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Menu from '@mui/material/Menu';
+import MenuIcon from '@mui/icons-material/Menu';
+import Container from '@mui/material/Container';
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import Tooltip from '@mui/material/Tooltip';
+import MenuItem from '@mui/material/MenuItem';
+import { Link } from 'react-router-dom';
+import Logo from '../svgs/logo';
+import './styles.css';
+
+export interface HeaderProps {}
+
+export interface MenuItemsInterface{
+  title: string,
+  href: string
+}
+
+export const MenuItems:MenuItemsInterface[] = [{
+  title: 'HOME',
+  href: '/'
+},
+{
+  title: 'SOBRE',
+  href: '/sobre'
+},
+{
+  title: 'AJUDA',
+  href: '/ajuda'
+},
+{
+  title: 'CONTATO',
+  href: '/contato'
+},
+{
+  title: 'AVALIAR O APP',
+  href: '/avaliar-app'
+},
+{
+  title: 'LOGIN',
+  href: '/login'
+}];
+
+export const Header: React.FC<HeaderProps> = (): JSX.Element => {
+  const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(
+    null,
+  );
+
+  const handleChangePage = (
+    target: React.MouseEvent<HTMLElement>['currentTarget'],
+  ) => {
+    setAnchorElNav(target);
+  };
+
+  const handleOpenNavMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorElNav(event.currentTarget);
+  };
+
+  const handleCloseNavMenu = () => {
+    setAnchorElNav(null);
+  };
+
+  return (
+    <AppBar position="static" className="app-header" elevation={0}>
+      <Container maxWidth="xl">
+        <Toolbar disableGutters className="toolbar">
+          <Typography
+            variant="h6"
+            noWrap
+            component="div"
+            sx={{ mr: 2, display: { xs: 'none', md: 'flex' } }}
+          >
+            <Logo/>
+          </Typography>
+
+          {/*MENU HAMBURGUER*/}
+
+          <Box sx={{ flexGrow: 1, display: { xs: 'flex', md: 'none' } }} >
+            <IconButton
+              size="large"
+              aria-label="account of current user"
+              aria-controls="menu-appbar"
+              aria-haspopup="true"
+              onClick={handleOpenNavMenu}
+              color="inherit"
+            >
+              <MenuIcon />
+            </IconButton>
+            <Menu
+              id="menu-appbar"
+              anchorEl={anchorElNav}
+              anchorOrigin={{
+                vertical: 'bottom',
+                horizontal: 'left',
+              }}
+              keepMounted
+              transformOrigin={{
+                vertical: 'top',
+                horizontal: 'left',
+              }}
+              open={Boolean(anchorElNav)}
+              onClose={handleCloseNavMenu}
+              sx={{
+                display: { xs: 'block', md: 'none' },
+              }}
+            >
+              {MenuItems.map((item, key) => 
+              <MenuItem key={key} to={item.href} component={Link}>
```

Comment:
Aqui eu vi que tanto o `MenuItem` quanto o `Typography` tem as propriedade `to` e `component` iguais (`item.href` e `Link`). Ta certo isso?? Será que se ficar desse jeito, não vai ficar algo parecido com isso no DOM:

```html
<a href=item.href>
    <a href=item.href>
        item.title
    </a>
</a>
```

Eu sei que provavelmente deve estar dando certo ne, já que afinal é um `a` normal. Mas será q isso não zoa as ferramentas de acessibilidade? Massss tlg que teve o problema com o teste ne, então não sei se da pra mudar.

Reply:
Oi Arthur, deixamos essas coisinhas passarem mesmo, estamos arrumando aqui!